rs/sns/governance/canister/governance_test.did

```diff
@@ -719,8 +719,15 @@ type WaitForQuietState = record {
   current_deadline_timestamp_seconds : nat64;
 };
 
+type RefreshCachedUpgradeStepsRequest = record {};
```

Comment:
I think it would be useful if in testing we could use this function to specify what the SNS-W should give, which would simplify integration tests. 

Currently, this just triggers an event that would otherwise be triggered by heartbeat, right?